source/funkin/play/notes/Strumline.hx

```diff
@@ -195,6 +195,11 @@ class Strumline extends FlxSpriteGroup
    */
   public function mayGhostTap():Bool
   {
+    if (!Preferences.ghostTapping)
+    {
+      return false;
+    }
```

Comment:
Forgot to add the define check here I think

Reply:
no the entire function its in already has the define

Reply:
oh ok